big_tests/tests/cets_disco_SUITE.erl

```diff
@@ -123,15 +204,34 @@ mock_timestamp(Node, Timestamp) ->
     ok = rpc(Node, meck, new, [mongoose_rdbms_timestamp, [passthrough, no_link]]),
     ok = rpc(Node, meck, expect, [mongoose_rdbms_timestamp, select, 0, Timestamp]),
     %% Ensure that we mock
-    EnsureMocked = fun() ->
-        Timestamp = rpc(Node, mongoose_rdbms_timestamp, select, [])
-        end,
-    EnsureMocked(),
-    [{ensure_mocked, EnsureMocked}, {mocked_timestamp, Timestamp}].
-
-ensure_mocked(Config) ->
-    EnsureMocked = proplists:get_value(ensure_mocked, Config),
-    EnsureMocked().
+    Timestamp = rpc(Node, mongoose_rdbms_timestamp, select, []).
```

Comment:
the comment is still valid since it was about execution of removed `EnsureMocked`, and we do the same, but without anonymous function.
anonymous function is not needed, because we now do mocking directly inside the tescase.
Timestamp is an input parameter, and it's used for mocking.